Woof/WoofTests/Model/Preferences/PreferencesHandlerLoadTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class PreferencesHandlerLoadTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        PreferencesHandlerTestHelper.cleanStorage()
+    }
+
+    override func tearDown() {
+        PreferencesHandlerTestHelper.cleanStorage()
+        super.tearDown()
+    }
+
+    func testLoadSelectedRoleExistInAPI() {
+        _ = PreferencesHandler.loadSelectedRole()
+    }
+
+    func testLoadSelectedRoleReturnsExpectedRole() {
+        _ = PreferencesHandler.saveSelectedRole(.owner)
+
+        XCTAssertEqual(PreferencesHandler.loadSelectedRole(), .owner)
+    }
+
+    func testLoadSelectedRoleReturnsDefaultCaseForNonExistingKey() {
```

Comment:
Which key doesn't exist?
I can't understand the intention of test from its name.

Reply:
i'll try to fix it

Reply:
Now the name is still confusing.
And you don't have to have markers "given-when-then" if your test is just a one line.

Reply:
Please pay attention what is a "given" section.

Wouldn't be the following test more robust:

```swift
// Given
let initialRole: Role = .owner
let overriddenRole: Role = .sitter
_ = PreferencesHandler.set(userRole: initialRole)

// When
_ = PreferencesHandler.set(userRole: overriddenRole)

// Then
XCTAssertNotEqual(PreferencesHandler.getRole(), initialRole)
XCTAssertEqual(PreferencesHandler.getRole(), overriddenRole)
```

?

Reply:
changed test function names here and elsewhere